ibm/service/eventnotification/resource_ibm_en_destination_custom_email.go

```diff
@@ -231,6 +284,12 @@ func resourceIBMEnCustomEmailDestinationUpdate(context context.Context, d *schem
 
 	options.SetInstanceID(parts[0])
 	options.SetID(parts[1])
+	verifyCustomEmailDestinationConfiguration.SetType(d.Get("verification_type").(string))
+	verifyCustomEmailDestinationConfiguration.SetInstanceID(parts[0])
+	verifyCustomEmailDestinationConfiguration.SetID(parts[1])
+	hasChangeverification := false
+
+	verifyCustomEmailDestinationConfiguration.SetType(d.Get("verification_type").(string))
 
 	if ok := d.HasChanges("name", "description", "collect_failed_events", "config"); ok {
```

Comment:
only when we have some changes on these fields we use `verification_type`?
During create we don't need this property

Reply:
Yes, it is only a update parameter, it's not needed  at the time of creation